pkg/pagerduty/service.go

```diff
@@ -362,6 +378,56 @@ func (c *SvcClient) DisableService(data *Data) error {
 	return nil
 }
 
+// ToggleServiceOrchestration enables/disables the service orchestration for a given PD service
+func (c *SvcClient) ToggleServiceOrchestration(data *Data, active bool) error {
```

Comment:
This is reasonable short-term, but should this be contributed upstream to https://github.com/PagerDuty/go-pagerduty/tree/main?

Reply:
I love this idea. :100:  I will put it on my list.

Reply:
https://issues.redhat.com/browse/OSD-13926 created here.